.github/workflows/l10n.yml

```diff
@@ -0,0 +1,954 @@
+name: L10n (Localization)
+
+# spell-checker: disable
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
+
+env:
+  # * style job configuration
+  STYLE_FAIL_ON_FAULT: true ## (bool) fail the build if a style job contains a fault (error or warning); may be overridden on a per-job basis
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+# End the current execution if there is a new changeset in the PR.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+
+  l10n_build_test:
+    name: L10n/Build and Test
+    runs-on: ${{ matrix.job.os }}
+    env:
+      SCCACHE_GHA_ENABLED: "true"
+      RUSTC_WRAPPER: "sccache"
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - { os: ubuntu-latest  , features: "feat_os_unix" }
+          - { os: macos-latest   , features: "feat_os_macos" }
+          - { os: windows-latest , features: "feat_os_windows" }
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+    - uses: dtolnay/rust-toolchain@stable
+    - uses: taiki-e/install-action@nextest
+    - uses: Swatinem/rust-cache@v2
+    - name: Run sccache-cache
+      uses: mozilla-actions/sccache-action@v0.0.9
+    - name: Install/setup prerequisites
+      shell: bash
+      run: |
+        ## Install/setup prerequisites
+        case '${{ matrix.job.os }}' in
+          ubuntu-*)
+            # selinux headers needed for testing
+            sudo apt-get -y update ; sudo apt-get -y install libselinux1-dev
+          ;;
+          macos-*)
+            # needed for testing
+            brew install coreutils
+          ;;
+        esac
+    - name: Build with l10n features
+      shell: bash
+      run: |
+        ## Build with l10n features
```

Comment:
"with l10n features" is a bit misleading as the features used with `--features` are not related to "l10n".